mimic/GUI/DebugLogWindow.py

```diff
@@ -23,6 +24,12 @@ def __init__(self, master: tk.Tk):
         super().__init__(master)
 
         self.master = master
+
+        # TkInter photoimage only supports PPM PGM image formats
+        # Using ImageTk and Image from PIL gets around this
+        mimic_logo = ImageTk.PhotoImage(Image.open("./assets/favicon.ico"))
+        self.iconphoto(False, mimic_logo)
```

Comment:
Instead of creating the icon and assigning it to each window twice, we can create and assign it once as the default in GUI.py